node/subsystem/src/lib.rs

```diff
@@ -219,9 +219,33 @@ pub trait SubsystemContext: Send + 'static {
 ///
 /// [`Overseer`]: struct.Overseer.html
 /// [`Subsystem`]: trait.Subsystem.html
+#[async_trait]
 pub trait Subsystem<C: SubsystemContext> {
+	/// This subsystem's name
```

Comment:
Worth noting that it only makes difference if the `start` method is implemented.